lib/Runtime/Types/ES5ArrayTypeHandler.cpp

```diff
@@ -371,7 +371,7 @@ namespace Js
         // Reject if we need to grow non-writable length
         if (!CanSetItemAt(arr, index))
         {
```

Comment:
There are 2 more similar fragments in this file ("if (!CanSetItemAt...) { return false; }"), please double check if they need fixes too.